ui/shared/utils/constants.js

```diff
@@ -1119,6 +1126,10 @@ export const PREDICTOR_FIELDS = [
   { field: 'metasvm', group: MISSENSE_IN_SILICO_GROUP, indicatorMap: INDICATOR_MAP },
   { field: 'gerp_rs', group: MISSENSE_IN_SILICO_GROUP, noSeverity: true, min: -13, max: 7 },
   { field: 'phastcons_100_vert', group: MISSENSE_IN_SILICO_GROUP, noSeverity: true },
+  { field: 'apogeeScore', warningThreshold: 0.5, dangerThreshold: 0.5, fieldTitle: 'APOGEE' },
+  { field: 'hapDefiningVariant', fieldTitle: 'Haplogroup Defining', indicatorMap: { Y: { color: 'green', value: '' } } },
```

Comment:
you should set the field in the elasticsearch config to return this with the desired display, `fieldTitle` is a last resort for things that need special casing. The other fields do have special casing so will need an explicit field title, but it would still be better for the field to better match the display (i.e. apogee instead of apogeeScore, mitotip instead of mitotipTrnaPrediction, htmvar instead of HmtVarScore)

Reply:
Okay, I can change the backend config.  If we use `apogee` as the response key, is it okay to display it as `Apogee`? How about MitoTIP and HmtVar, can they be displayed as `Mitotip` and `Hmtvar`?

Reply:
It seems that I can use 'mitoTIP' and 'hmtVar' for the response keys.